### ISSUE TITLE
Further scoping effort to refine exact requirements for each app interaction and development of robust ‘user stories’

### ISSUE THREAD

**Issue by drichardson23** [beak-lab/swell#5]
This is our first task, will be directed by Dan but whole team will need to be involved.

4 hour meeting, with an hour or two to document should cover this I think.

**Comment by drichardson23** [comment]
We didn't really do this given the tight timeframe, but I'd like to give it a go after the 7th.

**Comment by grgcnnr** [comment]
Agreed - would like to scope all the different inputs we're like;y to need and spend some time building widgets for each.

**Comment by grgcnnr** [comment]
Reassigning to alpha
